ath5k: Fix bad udelay calls on AR5210 code

Based on a patch by Nick Kossifidis <mick@madwifi.org>
http://git.kernel.org/?p=linux/kernel/git/davem/net-next-2.6.git;a=commitdiff;h=84e463fa0786a105c39281b90f8e3b6fe1444a05

--- a/drivers/net/wireless/ath5k/hw.c	2008-09-27 12:45:23.000000000 +0000
+++ b/drivers/net/wireless/ath5k/hw.c	2008-09-27 12:45:40.000000000 +0000
@@ -507,7 +507,7 @@
 	}
 
 	if (ah->ah_version == AR5K_AR5210)
-		udelay(2300);
+		mdelay(2);
 
 	/* ...wakeup again!*/
 	ret = ath5k_hw_set_power(ah, AR5K_PM_AWAKE, true, 0);
diff -urN drivers/net/wireless/ath5k~/phy.c drivers/net/wireless/ath5k/phy.c
--- a/drivers/net/wireless/ath5k/phy.c	2008-09-27 12:45:23.000000000 +0000
+++ b/drivers/net/wireless/ath5k/phy.c	2008-09-27 12:45:54.000000000 +0000
@@ -2001,7 +2001,7 @@
 	beacon = ath5k_hw_reg_read(ah, AR5K_BEACON_5210);
 	ath5k_hw_reg_write(ah, beacon & ~AR5K_BEACON_ENABLE, AR5K_BEACON_5210);
 
-	udelay(2300);
+	mdelay(2);
 
 	/*
 	 * Set the channel (with AGC turned off)
